common/pom.xml

```diff
@@ -226,7 +226,7 @@
 			<pluginRepositories>
 				<pluginRepository>
 					<id>reficio</id>
-					<url>http://repo.reficio.org/maven/</url>
+					<url>https://repo.reficio.org/maven/</url>
```

Comment:
Doesn't seem to work with https.

Reply:
@pzygielo Right, sorry for the inconvenience. I assumed that if the build passes the changes work, but that's not correct. Is the repository still needed? It won't work for long since Maven is dropping support for non HTTPS in the future. Maybe the plugin can be retrieved from another repo with HTTPS.

Reply:
Thanks for spotting that. The P2 plugin only gets used in a particular profile but it's important for making those libraries available in Eclipse and OSGi, so please check those things out in future for every proposed change.

Reply:
I think it's better to do #56. I also did https://github.com/krichter722/uom-lib/commit/064453771e7faf7cb92fa223b225efc968e78870 based on top of #56 and it works fine, so the repository might not be needed after all.

Reply:
I already fixed the URL in the POM before.

Reply:
@keilw I wanted to show that if I revert your revert it works as well which shows that the plugin can be retrieved from another location as well. I don't know what pzygielo tested. The URL doesn't work with https, though. For me it's fine to leave everything as it is now.